4/Dockerfile

```diff
@@ -19,10 +19,22 @@ WORKDIR /var/www/html
 RUN	apk add --no-cache libpq
 
 RUN	set -x \
+&&	apk add --no-cache --virtual .php-ext-deps \
+	unixodbc \
+	freetds \
 &&	apk add --no-cache --virtual .build-deps \
 	postgresql-dev \
 	sqlite-dev \
-&&	docker-php-ext-install pdo_mysql pdo_pgsql pdo_sqlite \
+	unixodbc-dev \
+	freetds-dev \
+&&	docker-php-ext-configure \
+	pdo_odbc --with-pdo-odbc=unixODBC,/usr \
```

Comment:
`docker-php-ext-configure` only supports passing a single extension, thus the line break is misleading here. Can you merge these two lines into a single one?